.github/pull_request_template.md

```diff
@@ -9,11 +9,21 @@
 ## Issue link
 <!-- Add a link to the issue -->
 
-## Checklist before requesting a review
-- [ ] I have performed a self-review of my code
-- [ ] I have added thorough, applicable tests that validate my changes
+## Pre-review checklist
+
+These should be followed unless you're dealing with an emergency. In an emergency, use your best judgment. 
+
+- [ ] I've tested the code as an end user would.
```

Comment:
Attempting to address @bryanspears 's comment about making this easy for reviewers to test. 

```suggestion
- [ ] I've tested the code as an end user would and included appropriate testing steps 
```

Reply:
Can you add a period on the end of this? I can't suggest a change to your suggested change ;)

Reply:
@bryanspears does that work for you? I like this change.